src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const reversedRobot = {};
+
+  for (const key in robot) {
+    const robotKey = robot[key];
```

Comment:
Seems like it doesnt have any scene to use this variable, it would only increase the time of your page loading☕